src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -84,8 +85,9 @@ void Background_Loaded(object sender, RoutedEventArgs e)
             Owner = _wndhost;
 
             _wndhost.Closing += Wndhost_Closing;
+            _wndhost.LocationChanged += RefreshOverlayPosition;
```

Comment:
Why on _wndhost and not on the background like other events?

Reply:
Cause _bckgnd has as type FrameworkElement no alternative for LayoutUpdated. So the only other choice to hook on some element is _wndhost.

Reply:
I've checked: "WindowsFormsHost" The actual type behind "FrameworkElement _bckgnd" also has no alternative. So even a cast would unfortunately not work